tracing/tracing.go

```diff
@@ -93,6 +95,21 @@ func InitTracer(opts []string) (tracer ot.Tracer, err error) {
 	}
 }
 
+// TryOtelTracerWithBridge tries to initialize a OpenTelemetry tracer and a
+// OpenTelemetry bridge tracer. If the user did not specify otel as tracer
+// implementation, it will just return nil for both tracers and no error.
+func TryOtelTracerWithBridge(ctx context.Context, opts []string) (opentelemetry.Tracer, ot.Tracer, error) {
+	if len(opts) == 0 {
+		return nil, nil, errors.New("tracing: the implementation parameter is mandatory")
```

Comment:
maybe better to `tracing` -> `otel.tracing` or so to differentiate the log to the code. It's always good to be able to grep the log in source code and get only one line that shows up and not more than one.

Reply:
I changed this, now everything is initialized in the same place and we only have one place that can produce this error.